tests/api/runtimes/base.py

```diff
@@ -354,15 +353,15 @@ def _generate_custom_object(
         ):
             return deepcopy(body)
 
-        get_k8s().crdapi.create_namespaced_custom_object = unittest.mock.Mock(
+        get_k8s_helper().crdapi.create_namespaced_custom_object = unittest.mock.Mock(
             side_effect=_generate_custom_object
         )
         self._mock_get_logger_pods()
 
     # Vault now supported in KubeJob and Serving, so moved to base.
     def _mock_vault_functionality(self):
-        secret_dict = {key: self.vault_secret_value for key in self.vault_secrets}
-        VaultStore.get_secrets = unittest.mock.Mock(return_value=secret_dict)
+        # secret_dict = {key: self.vault_secret_value for key in self.vault_secrets}
```

Comment:
I suggest adding a TODO with a mark like `# TODO: Vault:` where we commented out vault code so that it will be easier to find